SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1955,6 +1997,12 @@ class CVariable {
    */
   inline virtual CVectorOfMatrix& GetGradient_Reconstruction(void) { return Gradient; }
 
+  /*!
+   * \brief A virtual member.
+   * \return Value of the primitive variables gradient.
+   */
+  inline virtual su2double **GetAxiAuxVarGradient(unsigned long iPoint) { return nullptr; }
+
```

Comment:
Unless I'm missing something this does not have to be virtual?

Reply:
This was changed an inline only.   The "final" version in CEulersovler was removed.